src/sentry/static/sentry/app/views/settings/organizationSecurityAndPrivacy/organizationSecurityAndPrivacyContent.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import AsyncView from 'app/views/asyncView';
+import {GetEndPointsOutput} from 'app/components/asyncComponent';
+import {Organization} from 'app/types';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {updateOrganization} from 'app/actionCreators/organizations';
+import organizationSecurityAndPrivacy from 'app/data/forms/organizationSecurityAndPrivacy';
+
+import {OrganizationSecurityAndPrivacyProps} from './types';
+
+class OrganizationSecurityAndPrivacyContent extends AsyncView<
+  OrganizationSecurityAndPrivacyProps
+> {
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
+    router: PropTypes.object,
+  };
+
+  getEndpoints(): GetEndPointsOutput {
```

Comment:
Similar idea to `React.ComponentProps` we can use the component as the reference instead of explicitly importing/exporting the type:

```suggestion
ReturnType<AsyncView['getEndpoints']>
```